.github/workflows/generate-summary.yaml

```diff
@@ -86,6 +86,23 @@ jobs:
           init: false
           branch: ${{ inputs.branch }}
 
+      - name: Initialize binutils
```

Comment:
I think the only reason why we initialize gcc is because `download_artifacts.py` runs 
```
cd gcc && git checkout master --quite && git pull --quite && git rev-parse {git_hash}~10000
```
for artifact matching purposes. 
I'm not sure if we will be generating/regenerating new issues for binutils frequently. We may be able to ignore initialization (and the git server issues) by changing the requests url to include `labels=zve` and filter the `staging` label or whatever other labels out of the response. Unless we do a workflow dispatch with a older hash, everything should be in chronological order. We can also add a label signifying workflow dispatch that we want to ignore that hash for comparisons